packages/flutter/test/material/app_bar_test.dart

```diff
@@ -217,7 +217,7 @@ void main() {
     final Finder title = find.byKey(titleKey);
     expect(tester.getTopLeft(title).dx, 72.0);
     // The toolbar's contents are padded on the right by 4.0
-    expect(tester.getSize(title).width, equals(800.0 - 72.0 - 4.0));
+    expect(tester.getSize(title).width, equals(800.0 - 4.0 - 16.0 - 56.0 - 16.0));
```

Comment:
72.0 was a bunch of stuff previously bundled together. Was a bit too implicit. 
It was also previously missing a margin on the right side between the title and the actions. Readjusted slightly.